package.json

```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "build:clean": "rimraf ./build",
     "build": "cross-env NODE_ENV=production webpack --config internals/webpack/webpack.prod.babel.js --color --progress",
+    "generate": "plop --plopfile internals/generators/index.js",
```

Comment:
Nie działa,

 npm run generate                                                                                                                                               (3d1h27m) 16:37:13 

> react-ts-webpack@0.0.1 generate
> plop --plopfile internals/generators/index.js

sh: plop: command not found

Reply:
mialem plop jako globalny dependency. Dodalem do package.json